dags/notice_transformer_dag.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+sys.path.append("/opt/airflow/")
+sys.path = list(set(sys.path))
+import os
+
+os.chdir("/opt/airflow/")
```

Comment:
is this "hack" permanent? Or can we have something done at the "system" level?

Reply:
As I understood from Stefan, this is a requirement for DAG to run smooth in airflow environment